cpp/src_prims/sparse/csr.cuh

```diff
@@ -42,89 +42,63 @@ namespace sparse {
 
 struct WeakCCState {
  public:
-  bool *xa;
-  bool *fa;
   bool *m;
-
-  WeakCCState(bool *xa, bool *fa, bool *m) : xa(xa), fa(fa), m(m) {}
+  WeakCCState(bool *m) : m(m) {}
```

Comment:
We could probably snake case this too. Or I can do it when I pull it out to raft, either way works

Reply:
From Google style guide:

> Type names start with a capital letter and have a capital letter for each new word, with no underscores: MyExcitingClass, MyExcitingEnum.
> The names of all types — classes, structs, type aliases, enums, and type template parameters — have the same naming convention. Type names should start with a capital letter and have a capital letter for each new word.

I am renaming some variables that I forgot, though: `batch_size` and `start_vertex_id`

Reply:
I had originally pulled the weak cc code into separate prims and came up w/ the name `WeakCCState` before cuML was following any specific style standards. Actually, I still don't believe we are adhering to the Google standards across the codebase even though we list the Google style guide in the developer guide. 

I'm okay leaving it for now, but would like to revisit once this is moved over to raft because a lot of the class types in raft are snake cased, like `std_comms_t` and `mst` even though they are classes. It would be nice to strive for consistency whichever way we go.